composables/data/articlePages.ts

```diff
@@ -61,16 +78,13 @@ export function normalizeArticlePage(article: Record<string, any>): ArticlePage
     }
 }
 
+// normalize the author data
 function normalizeAuthor(author:Record<string, any>): Author {
     return {
-        id: author.id,
-        name: `${author.firstName} ${author.lastName}`,
-        photoID: author.photo,
-        jobTitle: author.jobTitle,
-        biography: author.biography,
-        website: author.website,
-        email: author.email,
-        slug: author.slug,
-        url: `/staff/${author.slug}`
+        firstName: author.firstName,
+        lastName: author.lastName,
+        url: author.slug && `/staff/${author.slug}`,
+        organization: author.contributingOrganization?.name,
+        organizationUrl: author.contributingOrganization?.url
```

Comment:
Why are we removing all this stuff from authors? I think we're going to need it for the author pages.

Reply:
I'll add them back in!